Rest/RestTests/ExtensionFunctionsTests.cs

```diff
@@ -41,6 +41,7 @@ public void ExtensionFunctionsIsLikeTest()
             Assert.IsTrue("abc".IsLike("a*"));
         }
 
+#if NET48
```

Comment:
We should be able to take the same approach for NET48 and NET50, and eliminate the need for TestData.xml